src/main/java/nextstep/courses/domain/ImageFileSize.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.courses.domain;
+
+public class ImageFileSize {
+
+    private static final int MAX_FILE_SIZE = 1024;
+    private static final String INVALID_IMAGE_FILE_SIZE = "이미지 사이즈는 1MB 이하만 가능합니다.";
+
+    private int imageFileSize;
```

Comment:
```suggestion
    private final int imageFileSize;
```

인스턴스 변수는 가능하면 final로 구현할 것을 추천